recipe/meta.yaml

```diff
@@ -73,3 +69,4 @@ about:
 extra:
     recipe-maintainers:
       - marcelotrevisani
+      - rcurtin
```

Comment:
@rcurtin , Do you want to maintain this recipe as well?

Reply:
@marcelotrevisani I appreciate the offer!  But I'd like to decline if that's okay :(  I already have too many other things I am maintaining, so I don't think I have the time to give this package the attention it needs.  I'm happy to help us work out the issues here in this PR, though, and then make sure our build system stays stable enough that the packaging job is easy. :)

Reply:
Sure @rcurtin ! No problem at all! I just thought that would be good for you as well, so you can modify the recipe and things like that. However, if you do not want/have time that is fine as well!
When I have any problem I will ping you then! :smile:

Reply:
Definitely, sounds good!